app/javascript/src/components/OnboardingUsers.jsx

```diff
@@ -78,6 +77,8 @@ class OnboardingUsers extends Component {
 OnboardingUsers.propTypes = {
   handleCheckUser: PropTypes.func.isRequired,
   handleCheckAllUsers: PropTypes.func.isRequired,
+  users: PropTypes.arrayOf(userPropType).isRequired,
+  checkedUsers: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
shouldn't this be an array of userproptype? or at least a shape

Reply:
fixed